src/components/profile/profile.js

```diff
@@ -0,0 +1,32 @@
+import { useSelector } from 'react-redux';
+import ReservedRockets from './ReservedRockest';
+import './Profile.css';
+
+function MyProfile() {
+  const missions = useSelector((state) => state.missions);
+  return (
+    <div className="container">
+      <section className="missions">
+        <h2>My Missions</h2>
+        <table className="missions-table">
+          <tbody>
+            {missions
+              && missions
+                .filter((mission) => mission.reserved === true)
+                .map((reserve) => (
+                  <tr className="row" key={reserve.missionId}>
+                    <td>{reserve.missionName}</td>
+                  </tr>
+                ))}
+          </tbody>
+        </table>
+      </section>
+      <section className="rockets">
+        <h2>My Rockets</h2>
+        <ReservedRockets />
+      </section>
+    </div>
```

Comment:
## Optional 

- [ ] The My Profile section can be enhanced by: 
-  adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved).
-  adding the "Cancel reservation" and "Leave Mission" buttons to the lists here. Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections